src/EditorFeatures/Test/EditAndContinue/EmitSolutionUpdateResultsTests.cs

```diff
@@ -50,7 +47,7 @@ public async Task GetHotReloadDiagnostics()
                     customTags: ImmutableArray.Create("Test2"),
                     properties: ImmutableDictionary<string, string?>.Empty,
                     document.Project.Id,
-                    new DiagnosticDataLocation("a.cs", document.Id, new TextSpan(1, 2), 0, 0, 0, 5, "a.razor", 10, 10, 10, 15),
+                    DiagnosticDataLocation.TestAccessor.Create(new("a.cs", new(0, 0), new(0, 5)), document.Id, new TextSpan(1, 2), new("a.razor", new(10, 10), new(10, 15)), checkMappedFileSpan: false),
```

Comment:
note: We have a test helper as some tests want to just directly create fake 'mapped spans' (e.g. a mapped span not actually backed by the data the compiler uses to truly state this is a mapped span).  our helper effectively says to trust these test-created values as if they were created by the compiler and thus are valid.